ontopy/ontology.py

```diff
@@ -108,16 +108,16 @@ def get_ontology(
 
         if base_iri == "emmo":
             base_iri = (
-                "http://emmo-repo.github.io/versions/1.0.0-beta4/emmo.ttl"
+                "http://emmo-repo.github.io/versions/1.0.0-beta5/emmo.ttl"
```

Comment:
We should not do this, but keep the latest stable version as standard.

Reply:
Fine for me. Then we can just drop this PR and skip testing get_indirect_is_a()